blog-admin-backend-org-page.el

```diff
@@ -38,6 +38,8 @@
 "
   "template for org-page's org post")
 
+(defvar config-file nil
```

Comment:
But, it is defined in the namespace and it is going to have the prefix. Right? 

I will define similar config variables for other backends.